### PR TITLE
MOL-144 Add endpoint to retrieve rendered mollie payment methods form

### PR DESCRIPTION
**Pull request by xNordo**

### CODE REVIEW

File: src/Controller/Action/Admin/MethodsFormAction.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+use Sylius\Bundle\PaymentBundle\Form\Type\PaymentMethodType;
+use Sylius\Component\Core\Model\PaymentMethod;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+
+
```
Comment (bartektw) [inline src/Controller/Action/Admin/MethodsFormAction.php:23]: Remove one empty line here

File: src/Controller/Action/Admin/MethodsFormAction.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+use Sylius\Bundle\PaymentBundle\Form\Type\PaymentMethodType;
+use Sylius\Component\Core\Model\PaymentMethod;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+
+final class MethodsFormAction extends AbstractController
```
Comment (daniel8702) [inline src/Controller/Action/Admin/MethodsFormAction.php:23]: Remove AbstractController

File: src/Controller/Action/Admin/MethodsFormAction.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    public function __construct(RepositoryInterface $gatewayConfigRepository, Environment $twig)
+    {
+        $this->gatewayConfigRepository = $gatewayConfigRepository;
+        $this->twig = $twig;
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $paymentCode = $request->get('paymentCode');
```
Comment (daniel8702) [inline src/Controller/Action/Admin/MethodsFormAction.php:39]: validate $paymentCode

File: src/Controller/Action/Admin/MethodsFormAction.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+        if (null === $paymentMethod) {
+            return new JsonResponse(['message' => sprintf("Payment method with code %s doesn't exist", $paymentCode)]);
+        }
+
+        $form = $this->createForm(PaymentMethodType::class, $paymentMethod);
+
+        $renderedForm = $this->twig->render(
+            'bundles/SyliusAdminBundle/PaymentMethod/_mollieMethodsForm.html.twig',
+            ['form' => $form->createView()]
```
Comment (daniel8702) [inline src/Controller/Action/Admin/MethodsFormAction.php:54]: formating

**Review by daniel8702 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**